sys/riotboot/flashwrite.c

```diff
@@ -122,10 +180,12 @@ int riotboot_flashwrite_finish_raw(riotboot_flashwrite_t *state,
         LOG_WARNING(LOG_PREFIX "re-flashing first block failed!\n");
         goto out;
     }
-
+#endif /* !CONFIG_RIOTBOOT_FLASHWRITE_RAW */
     LOG_INFO(LOG_PREFIX "riotboot flashing completed successfully\n");
     res = 0;
 
+#if !CONFIG_RIOTBOOT_FLASHWRITE_RAW
 out:
+#endif
```

Comment:
```suggestion
out:
```

Reply:
Your proposal results in an unused label error when using flashwrite_raw

Reply:
Ah I was hoping if we convert the proprocessor conditional to a  C if directive we wouldn't get that warning.

Reply:
Let me see what I can do :)

Reply:
I've rewritten the conditional and removed the goto+label. I haven't tested it, but it should be equivalent. Binary doesn't change in size, but the hash of the bin file is different between the two versions.